package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "e-referrals-service-api",
-  "version": "0.0.1",
+  "version": "v1.1.1-alpha",
```

Comment:
Probably worth leaving the package versions until a new version is cut from develop. I'll revert both this file and pyproject.toml here so they can be updated once v1.2.0 is created.